pulsar-common/src/test/java/org/apache/pulsar/common/util/FutureUtilTest.java

```diff
@@ -136,4 +139,37 @@ public void testGetOriginalException() {
                     assertTrue(future3Exception.get(0) instanceof IllegalStateException);
                 });
     }
+
+    @Test
+    public void testWaitForAny() {
```

Comment:
We should also cover the case that other futures could be canceled successfully before the waitForAny finished.

Reply:
right, fixed.